stdlib/test/builtin/test_bfloat16.mojo

```diff
@@ -35,46 +34,58 @@ def test_methods():
     assert_almost_equal(BFloat16(2.0), 2.0)
 
 
-fn test_bf_primitives():
+def test_bf_primitives():
     # we have to use dynamic values, otherwise these get evaled at compile time.
     var a = randn_float64().cast[DType.bfloat16]()
     var b = randn_float64().cast[DType.bfloat16]()
 
-    print(a + b)
-    print(a - b)
-    print(a / b)
-    print(a * b)
-    print(a == b)
-    print(a != b)
-    print(a <= b)
-    print(a >= b)
+    # higher precision
+    var a_hp = a.cast[DType.float64]()
+    var b_hp = b.cast[DType.float64]()
 
+    assert_almost_equal(a + b, (a_hp + b_hp).cast[DType.bfloat16]())
```

Comment:
Thanks for adding these!  Surprised the original code just had naked `print`s without any `CHECK` directives 🤷